src/parser/core/modules/Death.js

```diff
@@ -62,8 +62,9 @@ export default class Death extends Module {
 		}
 
 		if (!this.parser.fight.kill) {
-			// If the parse was a wipe, refund one death since the last one is pretty meaningless to ding them on
-			this._count--
+			// If the parse was a wipe, refund one death since the last one is pretty meaningless to ding them on.
+			// ...But max at 0 because apparently dummy parses don't get flagged as kills and -1 deaths makes very little sense.
+			this._count = Math.max(this._count - 1, 0)
```

Comment:
I opted for a refund rather than doing something with `shouldCountDeath()` mainly because this is logic tied to the fight as a whole, not to a specific event. There isn't really a clear way to say "this damage event caused the wipe so don't count the death right after it", especially in staggered wipes where half the party dies to one mechanic and causes the other half to die 10 seconds later to the next one as a result.